plugin/adsk/scripts/mayaUsdTranslatorImport.mel

```diff
@@ -140,7 +140,7 @@ global proc int mayaUsdTranslatorImport (string $parent,
         menuItem -l "Phong" -ann "phong";
 
         string $importUSDZTexturesAnn = "When a .usdz file is chosen and this is selected, .usdz texture files are imported and copied to new files on disk. To locate these files, navigate to the current Maya workspace /sourceimages directory.";
-        checkBoxGrp -label "USDZ Texture import: " -cw 1 $cw2 -ann $importUSDZTexturesAnn mayaUsdTranslator_ImportUSDZTexturesCheckBox;
+        checkBoxGrp -label "USDZ Texture Import: " -cw 1 $cw2 -ann $importUSDZTexturesAnn mayaUsdTranslator_ImportUSDZTexturesCheckBox;
```

Comment:
Actual fix. Above is another instance of "import" that I found when doing a search.